dbt_server/views.py

```diff
@@ -318,19 +363,31 @@ def get_task_status(
 
 
 class PostInvocationRequest(BaseModel):
-    # List of dbt command that will be sent to dbt worker for execution.
-    # E.g. ["--log-format", "json", "run", "--profiles_dir", "testdir"]
-    command: List[str]
-    # Dbt project path, if set --project-dir args will be appended into command
-    # list. If not set, dbt server will fallback to environment variable and
-    # append it. If environment variable is empty, request will be rejected.
+    # Dbt command that will be sent to dbt worker for execution, e.g. [
+    #   "--log-format", "json", "run", "--profiles_dir", "testdir"].
+    command: List[StrictStr]
```

Comment:
Originally missed this-- I'm not certain we want to limit this to strings. If this is to match the dbt-core command option types, we will get a list that includes strings as well as `int` and `list` types.
```suggestion
    command: List[Any]
```